browser/resources/brave_new_tab_page_refresh/components/app.style.ts

```diff
@@ -115,6 +115,10 @@ export const style = scoped.css`
     align-items: stretch;
     gap: 16px;
 
+    &:empty {
+      flex-basis: 0;
+    }
+
```

Comment:
nice pseudo class, and I assume we're doing it like this to avoid conditional rendering of the container in the TSX?

Reply:
Right, this is an alternative to checking widget visibility for all widgets in the `<App>` component (which it currently does not do).